exp-android-sdk/src/main/java/com/scala/exp/android/sdk/model/AbstractModel.java

```diff
@@ -61,9 +64,24 @@ public String toString() {
         return this.properties.toString();
     }
 
-
-    public void fling(IChannel channel, Map payload){
-        channel.fling(payload);
+    /**
+     * Get Channel with options
+     * @param options
+     * @return
+     */
+    public IChannel getChannel(Map options){
```

Comment:
instead of passing a Map, we should have 2 methods 
1) `public IChannel getChannel(boolean system, boolean consumer)` to match `Exp.getChannel(String, boolean, boolean)` 
2) 
```public IChannel getChannel(boolean system, boolean consumer) {
  return this.getChannel(false, false);
}````